src/Features/CSharp/Portable/TypeStyle/UseImplicitTypeCodeFixProvider.cs

```diff
@@ -38,16 +38,20 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             foreach (var diagnostic in diagnostics)
             {
                 var node = root.FindNode(diagnostic.Location.SourceSpan, getInnermostNodeForTie: true);
-
-                var implicitType = SyntaxFactory.IdentifierName("var")
-                                                .WithTriviaFrom(node);
-
-                editor.ReplaceNode(node, implicitType);
+                HandleDeclaration(editor, node);
```

Comment:
i almost feel like a cuter way to do thsi would be to just add the simplification annotation to the type, then the simplifier shoudl do this 😀 

But that's a lot of extra work that, at best, will have the same results as this (i hope).